lib/client/vm_api.rb

```diff
@@ -6,7 +6,11 @@ def vm(vm_id, opts={})
     end
 
     def vms(opts={})
-      headers = {:accept => "application/xml; detail=disks; detail=nics; detail=hosts"}
+      if opts[:without_details]
+        headers = {:accept => "application/xml"}
+      else
+        headers = {:accept => "application/xml; detail=disks; detail=nics; detail=hosts"}
+      end
       path = "/vms"
       path += search_url(opts) unless filtered_api
```

Comment:
@lzap thats fine, I  just learned that:
* oVirt admin account are not like our accounts, they are closer to roles that are tagged as admin (so you can give more people admin account, limited by its role)
* filtered API is really limited API and uses oVirt search internally (hence you can't use the search again in most / some cases).

Since foreman does not support filtered api, I don't see it as a problem for foreman, nor as a problem to rbovirt (as you will default to previous behaviour) so IMHO this is OK to merge.